djstripe/models/billing.py

```diff
@@ -1213,6 +1215,136 @@ def update_name(self):
         self.save()
 
 
+class Price(StripeModel):
+    """
+    Prices define the unit cost, currency, and (optional) billing cycle for
+    both recurring and one-time purchases of products.
+
+    Prices replace the old "Plan" model.
+
+    Stripe documentation:
+    - https://stripe.com/docs/api/prices
+    - https://stripe.com/docs/billing/prices-guide
+    """
+
+    stripe_class = stripe.Price
+    stripe_dashboard_item_name = "prices"
+
+    active = models.BooleanField(
+        help_text="Whether the price can be used for new purchases."
+    )
+    currency = StripeCurrencyCodeField()
+    nickname = models.CharField(
+        max_length=250,
+        blank=True,
+        help_text="A brief description of the plan, hidden from customers.",
+    )
+    product = StripeForeignKey(
+        "Product",
+        on_delete=models.CASCADE,
+        related_name="prices",
+        help_text="The product this price is associated with.",
+    )
+    recurring = JSONField(
+        null=True,
+        blank=True,
+        help_text=(
+            "The recurring components of a price such as `interval` and `usage_type`."
+        ),
+    )
+    type = StripeEnumField(
+        enum=enums.PriceType,
+        help_text=(
+            "Whether the price is for a one-time purchase or a recurring "
+            "(subscription) purchase."
+        ),
+    )
+    unit_amount = StripeQuantumCurrencyAmountField(
+        null=True,
+        blank=True,
+        help_text=(
+            "The unit amount in cents to be charged, represented as a whole "
+            "integer if possible. Null if a sub-cent precision is required."
+        ),
+    )
+    unit_amount_decimal = StripeDecimalCurrencyAmountField(
```

Comment:
Ah, I don't think we should use start uisng StripeDecimalCurrencyAmountField for cents, since everywhere else we've used it for dollars - #955 is already a nightmare, this would make it even worse.

I think a new field type for these decimal cents is needed? See #958

Reply:
I think it makes sense to winding back down to using it for those particular cases… and supporting, as a BWC hack, feeding int to it.

IMO it'd be even more confusing to introduce a *third* field type. This legitimizes the usefulness of both field types.